clarity-ext-scripts/requirements.txt

```diff
@@ -3,4 +3,5 @@ mock==3.0.5
 pytest==4.6.9
 requests[socks]==2.23.0
 retry==0.9.2
-xlwt==1.3.0
+suds==0.4
```

Comment:
I see that this is imported, but I can't see it being used in this code. I guess it was missing from a previous PR?

Reply:
We're using suds in the SmiNet client. It's for SOAP.

Reply:
Used for soap (in the sminet_client)